allauth/usersessions/signals.py

```diff
@@ -1,8 +1,14 @@
+from django.dispatch import Signal
+
 from allauth.account import app_settings
 
 from .models import UserSession
 
 
+user_agent_changed = Signal()
+ip_changed = Signal()
```

Comment:
Can you please add these to `docs/usersessions/signals.rst` just like `docs/account/signals.rst` ?